raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -61,20 +62,26 @@ def cancel_other_transfers(payment_state: InitiatorPaymentState) -> None:
         initiator_state.transfer_state = "transfer_cancelled"
 
 
-def can_cancel(initiator: InitiatorTransferState) -> bool:
+def can_cancel(initiator: Optional[InitiatorTransferState]) -> bool:
```

Comment:
Should we really handle the `None` case here? It seems counterintuitive to me to ask "Can I cancel this non-existing transfer"?